scripts/before_ci_build.sh

```diff
@@ -1,23 +1,27 @@
+GMP_VERSION=6.2.1
+MPFR_VERSION=4.1.0
+MPC_VERSION=1.2.1
 if [ ! -f finish_before_ci_build ]; then
-  if [[ "$OSTYPE" == "linux-gnu" ]]; then
-    # yum install -y wget lzip
-    # wget https://gmplib.org/download/gmp/gmp-6.2.0.tar.lz
-    # tar -xvf gmp-6.2.0.tar.lz
+  if [ "$OSTYPE" == "linux-gnu" -o "$OSTYPE" == "darwin"* ]; then
```

Comment:
`OSTYPE` is not set in CI. Can you do `$(uname) == "Darwin"`?

Reply:
This variable was from bash, which run the before_ci_build.sh.  Why we can't use one?

BTW, does this pr fix #310?

Reply:
> BTW, does this pr fix #310?

No, because this condition is not met.

Reply:
Indeed, wildcards don't work for the POSIX test utility.  Bashism restored.